src/mui/components/textField/BasicTextField.tsx

```diff
@@ -35,6 +36,7 @@ function BasicTextField({
 
     return (
         <TextField
+            className={`text-field-${s.slugify(label)}`}
```

Comment:
Let's just get rid of this component https://github.com/Exabyte-io/web-app/pull/2375/files#r1411477268